README.md

```diff
@@ -1,6 +1,16 @@
 # Bevy Quickstart
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io). You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+The template brought to you by the Bevy Jam working group.
+
+This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game — especially for a game jam!
+Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
+You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+
+## Prerequisites
+
+We assume that you know how to use Bevy already and have at least gone through the [official Quick Start Guide](https://bevyengine.org/learn/quick-start/introduction/).
+If you are new to Bevy, the patterns used throughout the template may look a bit weird at first glance.
```

Comment:
```suggestion
```
I'd say this sentence is redundant.

Reply:
I want to make reassure people who are new. I know that looking at someone else's codebase can be a bit overwhelming at first, especially if you get the vibe that you're supposed to understand what's going on.